power_grid_model_c/power_grid_model_cpp/include/power_grid_model_cpp/handle.hpp

```diff
@@ -71,8 +71,8 @@ class Handle {
             auto const* const failed_scenario_seqs = PGM_failed_scenarios(handle_ptr);
             auto const* const failed_scenario_messages = PGM_batch_errors(handle_ptr);
             for (Idx i = 0; i < n_failed_scenarios; ++i) {
-                failed_scenarios[i] = PowerGridBatchError::FailedScenario{.scenario = failed_scenario_seqs[i],
-                                                                          .error_message = failed_scenario_messages[i]};
+                failed_scenarios[i] =
+                    PowerGridBatchError::FailedScenario{failed_scenario_seqs[i], failed_scenario_messages[i]};
```

Comment:
designated initializers are a C++20 feature. Clang-tidy-19 on Windows is OK with this change